kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/filtered-resources/kjars-sources/cloudbalance/src/main/java/org/kie/server/testing/CloudBalancingGenerator.java

```diff
@@ -50,31 +51,73 @@ public int getCost() {
     }
 
     private static final Price[] CPU_POWER_PRICES = { // in gigahertz
-                                                      new Price( 3, "single core 3ghz", 110 ),
-                                                      new Price( 4, "dual core 2ghz", 140 ),
-                                                      new Price( 6, "dual core 3ghz", 180 ),
-                                                      new Price( 8, "quad core 2ghz", 270 ),
-                                                      new Price( 12, "quad core 3ghz", 400 ),
-                                                      new Price( 16, "quad core 4ghz", 1000 ),
-                                                      new Price( 24, "eight core 3ghz", 3000 ),
+            new Price(3,
+                      "single core 3ghz",
+                      110),
+            new Price(4,
+                      "dual core 2ghz",
+                      140),
+            new Price(6,
+                      "dual core 3ghz",
+                      180),
+            new Price(8,
+                      "quad core 2ghz",
+                      270),
+            new Price(12,
+                      "quad core 3ghz",
+                      400),
+            new Price(16,
+                      "quad core 4ghz",
+                      1000),
+            new Price(24,
+                      "eight core 3ghz",
+                      3000),
     };
     private static final Price[] MEMORY_PRICES = { // in gigabyte RAM
-            new Price(2, "2 gigabyte", 140),
-            new Price(4, "4 gigabyte", 180),
-            new Price(8, "8 gigabyte", 220),
-            new Price(16, "16 gigabyte", 300),
-            new Price(32, "32 gigabyte", 400),
-            new Price(64, "64 gigabyte", 600),
-            new Price(96, "96 gigabyte", 1000),
+            new Price(2,
+                      "2 gigabyte",
+                      140),
+            new Price(4,
+                      "4 gigabyte",
+                      180),
+            new Price(8,
+                      "8 gigabyte",
+                      220),
+            new Price(16,
+                      "16 gigabyte",
+                      300),
+            new Price(32,
+                      "32 gigabyte",
+                      400),
+            new Price(64,
+                      "64 gigabyte",
+                      600),
+            new Price(96,
+                      "96 gigabyte",
+                      1000),
```

Comment:
My eyes are bleeding :) but ignore this comment, it's better to have a style that everyone follows than to bikeshed this further :)